CHANGES

```diff
@@ -44,6 +44,15 @@
      and L<EVP_MAC_final(3)>.
      [Paul Dale]
 
+  *) Over two thousand fixes were made to the documentation, including:
+     common options (such as -rand/-writerand, TLS version control, etc)
+     were refactored and point to newly-enhanced descriptions in openssl.pod,
+     documented all missing options, added a CI build target to make
+     sure all options are documented, have style conformance (with help
+     from Richard Levitte), documented many internals, and addressed all
+     internal broken L<> references.
+     [Rich Salz]
```

Comment:
IMHO this long paragraph deserves to be broken up into enumeration items. Also, I made a little logical change: you didn't document all missing options, you added documentation for all options which were missing it.

```
  *) Over two thousand fixes were made to the documentation, including:
     - Common options (such as -rand/-writerand, TLS version control, etc)
       were refactored and point to newly-enhanced descriptions in openssl.pod.
     - Added a CI build target to make sure all options are documented and have
       style conformance (with help from Richard Levitte), and documented all
       reported missing options.
     - Added a CI build target to make sure all options are documented and have
       style conformance (with Richard Levitte), and added documentation for all
       options which were missing it.
     - Documented many internals.
     - Addressed all internal broken L<> references.
     [Rich Salz]
```

_(just a suggestion, not a change request)_

Reply:
> BTW: does find-doc-nits also report if too many options are documented? 

Yes it reports "documented but not implemented"; that's how I found out about "hmac" changing, commit 3ed8675ee66b7c27ec40873ea614c376077fe2ee.

As for the other change, shrug.  I wanna get rid of the "over 2000" since it's closer to 2500 now and I want to avoid humble brags.  Let's see what others think.

Reply:
In order to reduce possible RTT's :) I added a fixup commit that basically does your suggestions.